massa-protocol-worker-2/src/handlers/endorsement_handler/commands_propagation.rs

```diff
@@ -1,6 +1,7 @@
 use massa_storage::Storage;
 
-pub enum EndorsementHandlerCommand {
+pub enum EndorsementHandlerPropagationCommand {
```

Comment:
Think the previous name is better with "stop"

```suggestion
pub enum EndorsementHandlerCommand {
```

Reply:
Ow ok , because you have add : EndorsementHandlerRetrievalCommand

Reply:
Yes I specified everywhere now that we have command in both threads